Arthur Zarins Assignment 1

azarins1 opened the pull request:
Assignment #1

atozprogramming committed:
First commit message

Commit:
9/12/2019 @ 5:19 PM

Commit:
Completed the Chapter 1 assignment:
ch_1_assign_Arthur_Zarins_.py

Commit:
Turn-in Assignment

The assignment is complete, with all of the events and stats for
the teams. The program has proper grammer, and answers
3 sample questions

Commit:
Submit Assignment

improve Grammer

Commit:
Turned-in assignment:
Input, and comments

Commit:
Turn-in Assignment

Ethan-Talbert reviewed (commented):
Bad print statements

Commit:
Chapter 1 assignment revised

Commit:
Assignment 2 syntax improved

Commit:
The Island minigame (fun project) has been
debugged! It's redy for gameplay

Commit:
Chapter 2 complete

Commit:
Chapter 3 assignment (w/ input validation)
 completed

Commit:
Proper data input types, Chapter 3 assignment
 improved

Commit:
Great Chapter 3 overwriting-prevention function

Commit:
Chapter 4 completed

Commit:
Added a bonus project: Rubik's cubes!
This program tells you the number of combonations
(pardon the spelling) from a 2x2 to Infinity x Infinity
cubes! Have fun learning how the cubes progressively
become more advanced! :)

Commit:
Added clarification comments + better code
for assignments 1 and 2

Commit:
Updated chapter 4

Commit:
.

Commit:
Added user input

Commit:
Changed a for loop

Commit:
Added a 5th team to Assignment 1

Commit:
runs

Commit:
Fixed chapter 1

Commit:
Added competition locations

Commit:
Fix chapter 1 and 2's running errors (Running when
imported or not running at all when called)

Commit:
Allow team names to be used to issue commands
